proposals/csharp-12.0/collection-expressions.md

```diff
@@ -158,21 +161,33 @@ namespace System.Runtime.CompilerServices
 The attribute can be applied to a `class`, `struct`, `ref struct`, or `interface`.
 The attribute is not inherited although the attribute can be applied to a base `class` or an `abstract class`.
 
-The collection type must have an [*iteration type*](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/statements.md#1295-the-foreach-statement).
+The *builder type* must be a non-generic `class` or `struct`.
 
-For the *create method*:
+First, the set of applicable *create methods* `CM` is determined.  
+It consists of methods that meet the following requirements:
 
-* The *builder type* must be a non-generic `class` or `struct`.
+* The method must have name specified in the `[CollectionBuilder(...)]` attribute. 
```

Comment:
total nit: but for ultra pedantry, we probably want to say that the method is not an explicit-impl.  but maybe that falls out since an explicit impl method would not be accessible.

Reply:
That lapse seems to predate this PR. I also noticed that the current speclet doesn't mention an substitution for the create method. Those should probably be addressed separately.

Reply:
I think, we do not care if the method explicitly implements anything as long as it is accessible. And yes, such methods exist in metadata. What we, perhaps, should care about that is whether the name is a valid identifier.